### PR TITLE
feat: suppress send welcome message

### PR DESCRIPTION
**Pull request by fabio-andre-rodrigues**

### CODE REVIEW

File: src/targets/adminCreateUser.ts [hunk 1]
```diff
@@ -105,7 +105,13 @@ export const AdminCreateUser =
   async (ctx, req) => {
     const userPool = await cognito.getUserPool(ctx, req.UserPoolId);
     const existingUser = await userPool.getUserByUsername(ctx, req.Username);
-    if (existingUser && req.MessageAction === "RESEND") {
+    const supressWelcomeMessage =
+      req.MessageAction === "SUPRESS";
+
+    const resendWelcomeMessage =
```
Comment (venkyhodigere) [inline src/targets/adminCreateUser.ts:111]: Do we need the resend message constant here? We only dealing with the suppress message. Maybe remove this and add it when we are ready for resend message functionality.

File: src/targets/adminCreateUser.ts [hunk 2]
```diff
@@ -105,7 +105,13 @@ export const AdminCreateUser =
   async (ctx, req) => {
     const userPool = await cognito.getUserPool(ctx, req.UserPoolId);
     const existingUser = await userPool.getUserByUsername(ctx, req.Username);
-    if (existingUser && req.MessageAction === "RESEND") {
+    const supressWelcomeMessage =
+      req.MessageAction === "SUPRESS";
```
Comment (venkyhodigere) [inline src/targets/adminCreateUser.ts:109]: SUPRESS is typoed. Need to be spelt "SUPPRESS"

**Review by venkyhodigere — COMMENTED** [review]
Need to add unit tests (by updating the spec file)

File: src/targets/adminCreateUser.ts (hunk 1 above)
Comment (fabio-andre-rodrigues) [reply src/targets/adminCreateUser.ts:111]: `if (existingUser && resendWelcomeMessage) {`

i just created a var for the check in the line 108

if (existingUser && req.MessageAction === "RESEND") {

so it reads better

File: src/targets/adminCreateUser.ts (hunk 2 above)
Comment (fabio-andre-rodrigues) [reply src/targets/adminCreateUser.ts:109]: thx

**Review by venkyhodigere — COMMENTED** [review]
Check if the integration tests here needs update. 
https://github.com/jagregory/cognito-local/blob/master/integration-tests/aws-sdk/adminCreateUser.test.ts

**Pull request merged**